openpype/hosts/fusion/plugins/create/create_saver.py

```diff
@@ -206,6 +220,9 @@ def get_pre_create_attr_defs(self):
         attr_defs = [
             self._get_render_target_enum(),
             self._get_reviewable_bool(),
+            BoolDef(
+                "custom_range", label="Custom range", default=False,
+            )
```

Comment:
Can we do this in different PR please? Would you be ok to create an issue for this?

Reply:
> Can we do this in different PR please? 

I'd say no, because by then we'd be breaking backwards compatibility since the setting key changes. If you want the least effort for this PR then turn this into an enum but skip the "Custom instance frame range" entry in the enum def.

Reply:
> Use comp range

Still dont understand why do you need this option, since the comp sould be set to asset framerange - isnt this duplicity? I am assuming User is using menu option for setting asset framerange onto the comp and then why whould you need to have such an option? 

> Use custom instance frame range

This is something which should be definable with the use of instance attributes - but it will be pretty confusing to users if they will always see the `frame in` and `frame out` in attributes, even while they select Asset attributes in enumerator - I am assuming you realize we do not have interactivity in UX and those Frame In and Out inputs are not hiddable?

Reply:
I had renamed the attribute key to `viewer_render_range` so I hope it makes more sense now. This will allow users to set similar behaviour as it was before.

Reply:
Ok the 3 ways enumerator options are added. The option `From composition timeline` is preserving the behavior exactly as it was before.